bazel_6.bazelrc

```diff
@@ -7,6 +7,3 @@ build --enable_bzlmod
 build --apple_crosstool_top=@local_config_apple_cc//:toolchain
 build --crosstool_top=@local_config_apple_cc//:toolchain
 build --host_crosstool_top=@local_config_apple_cc//:toolchain
-
-# Disabled until bugs are fixed
```

Comment:
The bugs were fixed, I guess?

Reply:
It's disabled by default in 6.4.0, so no need to call it out here. Didn't want to see if they actually were fixed and enable it.